sale_pricelist_auto_update/models/res_partner.py

```diff
@@ -97,7 +102,7 @@ def reset_partner_pricelist(self, date_range, partner_id=False):
         sql = """
             SELECT
                 p.commercial_partner_id AS partner_id,
-                SUM(base_amt) AS amount
+                SUM(sol.base_amt) AS amount
```

Comment:
Not necessary unless `base_amt` appears in more than one table.  But it's good to be explicit.